app/assets/javascripts/components/Register.jsx

```diff
@@ -0,0 +1,82 @@
+class Register extends React.Component {
+
+  FieldGroup({ id, label, help, ...props }) {
+    const { Checkbox, Radio, FormGroup, ControlLabel, FormControl, Button, HelpBlock } = ReactBootstrap;
+    return (
+      <FormGroup controlId={id}>
+        <ControlLabel>{label}</ControlLabel>
+        <FormControl {...props} />
+        {help && <HelpBlock>{help}</HelpBlock>}
+      </FormGroup>
+    );
+  }
+
+  createUser = (evt) => {
+    evt.preventDefault()
+    const inputs = evt.target.getElementsByTagName("input");
+    let upload = "";
+    if (inputs[4].files.length > 0) {
+      upload = inputs[4].files[0];
+    }
+    const payload = {
+      "email": inputs[0].value,
+      "password": inputs[1].value,
+      "first_name": inputs[2].value,
+      "last_name": inputs[3].value,
+      "avatar": upload
+    };
+    console.log(payload);
+    Requester.post('/api/users', payload).then((data) => {
+      // Handle success/failure.
```

Comment:
could you show an error message on failure? and a success message otherwise

Reply:
Just added it; let me know if it still needs further changes, or if I'm doing it in an unidiomatic way.